streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -483,6 +552,79 @@ public static void waitUntilMetadataIsPropagated(final List<KafkaServer> servers
 
     }
 
+    public static <K, V> void verifyOutput(final String broker,
+                                           final String topic,
+                                           final Deserializer<K> keyDeserializer,
+                                           final Deserializer<V> valueDeserializer,
+                                           final List<KeyValueTimestamp<String, Long>> expected) {
```

Comment:
Same here with properties.  
Also, do we want to have a generic `List<K, V>` we are providing to verify or `KeyValueTimestamp<K, V>`?

If we keep it as `KeyValueTimestamp maybe change the name to reflect it's not for verifying generic output.

Reply:
I'll change the name, since the verification specifically checks the key, value, and timestamp.